assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -47,8 +47,12 @@ export default function useMediaPicker({
   } = useConfig();
   const { showSnackbar } = useSnackbar();
   useEffect(() => {
-    // Work around that forces default tab as upload tab.
-    wp.media.controller.Library.prototype.defaults.contentUserSetting = false;
+    try {
+      // Work around that forces default tab as upload tab.
+      wp.media.controller.Library.prototype.defaults.contentUserSetting = false;
+    } catch (e) {
+      // Silence.
```

Comment:
Will this be in the final version?

Reply:
Probably not. But I think this will become clear with the editor/WP separation. We won't be able to keep this code here anyway, so instead this should be some sort of an extension point.